api/src/main/java/org/openmrs/module/rwandahivflowsheet/impl/pih/ProphylaxisMapping.java

```diff
@@ -111,23 +112,20 @@ public boolean getProphylaxisOther() {
 		public String getProphylaxisTypeOther(){
 			if (drugOrder != null && drugOrder.getConcept() != null 
 					&& !isCotrimoxisole() && !isFluconazole() && !isDapsone())
-				return drugOrder.getConcept().getBestName(Context.getLocale()).getName();
+				return drugOrder.getConcept().getDisplayString();
 			return "";
 		}
 		
 		public Date getStopDate(){
 			if (drugOrder != null){
-				if (drugOrder.getDiscontinuedDate() != null)
```

Comment:
Are we intentionally ignoring the discontinuedDate check?

Reply:
discontinuedDate is no longer available, so I'm using #getEffectiveStopDate which from the javadoc seems to handle this efficiently

Reply:
This seems right to me.